03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,53 @@
-<!DOCTYPE html>
-<html>
-    <head>
-        <title>The Daily Planet</title>
-        <link rel="stylesheet" href="css/daily-planet-style.css"/>
-    </head>
-    <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>The Daily Planet</title>
+    <link rel="stylesheet" href="css/daily-planet-style.css" />
+</head>
+
+<body>
+    <header>
+        <h1 >The Daily Planet</h1>
+        <nav>
+            <a href="http://alpinedailyplanet.typepad.com/alpine-daily-planet/sports/"> |Sports| </a>
+            <a href="http://kxan.com/category/news/local/"> |Local| </a>
+            <a href="http://www.bloomberg.com/live"> |Economy| </a>
+        </nav>
+    </header>
+
+    <main>
+        <h2>Pokemon Gooooo!</h2>
+        <h5>Pokémon Go tops $160M in global revenue, time spent in-app remains high</h5>
+        <h6>Posted 1 hour ago by Darrell Etherington (@etherington)</h6>
+        <hr></hr>
+        <img src="https://tctechcrunch2011.files.wordpress.com/2016/08/28285207320_c0a14460b9_k.jpg?w=1318"height="300px"width="500px" >
+        <section>
+        <p>I’m still trying to deal with losing Pokévision, too, but Niantic’s recent changes to the game don’t seem to be putting a damper on player appetite just yet — new data from analytics firm Sensor Tower reveals that users are still spending, on
```

Comment:
The only project requirement you are missing is the "article" tag. If you can update this and commit it again, I can bump your grade up to 100. Overall, great job! You are really getting the hang of this.